src/db/models/request.js

```diff
@@ -1,19 +1,17 @@
 
 module.exports = (sequelize, DataTypes) => {
   const Request = sequelize.define('Request', {
-    userId: DataTypes.INTEGER,
+    managerId: DataTypes.INTEGER,
+    requesterId: DataTypes.INTEGER,
+    origin: DataTypes.STRING,
+    destination: DataTypes.STRING,
     type: DataTypes.STRING,
+    departureDate: DataTypes.DATEONLY,
+    returnDate: DataTypes.DATEONLY,
     reason: DataTypes.STRING,
-    destination: DataTypes.STRING,
     status: DataTypes.STRING,
-    departureDate: DataTypes.DATE,
-    returnDate: DataTypes.DATE
   }, {});
   Request.associate = (models) => {
-    Request.belongsTo(models.User, {
```

Comment:
Why has this association been removed? It may be needed.

Reply:
Hello @MuhanguziDavid ,
I have created a migration to update request tables and it is where I have created the required associations.